recipes/libuv/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class libuvConan(ConanFile):
+    name = "libuv"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libuv.org"
+    description = "A multi-platform support library with a focus on asynchronous I/O"
+    topics = ("libuv", "asynchronous", "io", "networking", "multi-platform", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
```

Comment:
Why not fPIC in options?

Reply:
Will be added if needed, hooks treat it as a warning, but if there is a need, I will fix that

Reply:
Done: d5263911a93d512afe0cae6fdde8d90795139332